django/core/management/commands/dumpdata.py

```diff
@@ -6,6 +7,39 @@
 from django.core.management.utils import parse_apps_and_model_labels
 from django.db import DEFAULT_DB_ALIAS, router
 
+try:
+    import bz2
+    has_bz2 = True
+except ImportError:
+    has_bz2 = False
+
+try:
+    import lzma
+    has_lzma = True
+    lzma_kwarg = {'format': lzma.FORMAT_ALONE}
+except ImportError:
+    has_lzma = False
+
+
+def get_open_method(filename, mode='wt'):
```

Comment:
`t` is unnecessary, IMO.

Reply:
`t` is for text mode in `open` methods. (e.g. in [`gzip.open`](https://docs.python.org/3/library/gzip.html#gzip.open) )

Reply:
True, thanks :+1: